Formula/c/charmcraft.rb

```diff
@@ -17,20 +17,26 @@ class Charmcraft < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "7d6ce9798ec325abab4604c95e8754ce5872e2563b22cd4e43a4560c4cbae358"
   end
 
-  depends_on "rust" => :build # for rpds-py
-  depends_on "cffi"
-  depends_on "keyring"
+  depends_on "rust" => :build
   depends_on "libsodium"
-  depends_on "pycparser"
+  depends_on "libyaml" # for rpds-py
   depends_on "python-certifi"
   depends_on "python-cryptography"
```

Comment:
Just realized I'm specifying this depends_on twice, both here and within on_linux